lib/events.js

```diff
@@ -97,7 +97,12 @@ $(document).ready(function() {
             name: $newFoodName.val(),
             calories: $newFoodCalories.val()
         }
-        Food.addNewFood(food, $foodIndex)
+        if(Food.validate(food)) {
+          Food.addNewFood(food, $foodIndex);
+          $newFoodName.val('');
+          $newFoodCalories.val('');
```

Comment:
you could break out clearing the fields into a smaller function so you could reuse this in other places (if you need it of course) 👍